x-pack/plugins/ingest_manager/server/plugin.ts

```diff
@@ -11,7 +11,9 @@ import {
   Plugin,
   PluginInitializerContext,
   SavedObjectsServiceStart,
-} from 'src/core/server';
+  RecursiveReadonly,
+} from 'kibana/server';
+import { deepFreeze } from '../../../../src/core/utils';
```

Comment:
👍 did not know about this 😃

Reply:
Yeah I didn't either haha, I found it in a couple other apps.